crates/re_types/src/image.rs

```diff
@@ -3,8 +3,11 @@ use smallvec::{smallvec, SmallVec};
 use crate::datatypes::{TensorData, TensorDimension};
 
 #[derive(thiserror::Error, Clone, Debug)]
-pub enum ImageConstructionError<T: TryInto<TensorData>> {
-    #[error("Could not convert source to TensorData")]
+pub enum ImageConstructionError<T: TryInto<TensorData>>
+where
+    T::Error: std::fmt::Debug,
+{
+    #[error("Could not convert source to TensorData: {0:?}")]
```

Comment:
Why `Debug`? I'd expect `T::Error` to implement `std::error::Error` which would imply `std::fmt::Display`?

Reply:
Ahh, let me see if I can make that bound work.